Composer/packages/client/src/recoilModel/dispatchers/storage.ts

```diff
@@ -155,6 +156,43 @@ export const storageDispatcher = () => {
     }
   });
 
+  const fetchTemplatesV2 = useRecoilCallback(({ set }: CallbackInterface) => async (feedUrls?: string[]) => {
```

Comment:
prefer all of this to be in the component itself - it doesn't need to be in app-wide state, right? It is only used in the one part of the create new wizard flow.

Reply:
If we are getting rid of the template view in the home.tsx view then yeah it only needs to be in the createOptions component.

Reply:
OK good point. We can leave that in place for now, but when/if the homepage gets revamped, we should move this.